scripts/check_docs

```diff
@@ -10,16 +10,22 @@ source "$SCRIPTS_DIR/common"
 # false positive warning removed (https://github.com/rust-lang/rust/pull/70789)
 export RUSTDOCFLAGS="-A unused_braces"
 
-DOCS_OUT="$(cargo doc --document-private-items --no-deps 2>&1)"
+paths=("." "oak_abi" "../oak_utils")
```

Comment:
Having paths that are relative to the previous path in the list is confusing; probably better to have all the paths relative to the top of the repo and put something like `cd -` at the bottom of each loop iteration.

Reply:
[style](https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names): `readonly PATHS=...`   (Also, I'm not sure whether an explicit `declare -a` is preferred or not.)